haskell-src/Concordium/Types/Execution.hs

```diff
@@ -557,6 +566,12 @@ instance AE.ToJSON Payload where
               "proofAggregation" AE..= ubkProofAggregation,
               "transactionType" AE..= AE.String "updateBakerKeys"
             ]
+    toJSON TokenHolder{..} =
+        AE.object
+            [ "tokenSymbol" AE..= thTokenSymbol,
+              "operations" AE..= thOperations,
```

Comment:
Since we want to pass the `operation` via a JSON file from `concordium-client`. Suggestion would be to have some wrapper for that parsing, similar to:

https://github.com/Concordium/concordium-base/blob/e3672c71aba0fb06eb2fc0951b86367845d5e4d7/haskell-src/Concordium/Types.hs#L1184


```suggestion
              "operations" AE..= EncodedTokenOperations thOperations,
```

Reply:
I don't have a complete overview of what relies on this implementation, but I sounds like a good idea to have and should be made in a way that is compatible with the current implementation.
Maybe for another PR though, so I have created [this task](https://linear.app/concordium/issue/COR-1336)